Project/Assets/Editor/CustomInspectors/PromptEditor.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using UnityEditor;
+
+[CustomEditor(typeof(Prompt))]
+public class PromptEditor : Editor {
+
+	Prompt prompt;
+
+	public override void OnInspectorGUI ()
+	{
+		prompt = (Prompt)target;
+
+		GUIContent label = new GUIContent ("Prompt Text", "Text displayed when a player can interact with this object.");
+		prompt.promptText = EditorGUILayout.TextField (label, prompt.promptText);
+
+		if (prompt.promptText == null || prompt.promptText == "")
+		{
+			// I feel like there should be a warning here, but I don't know what to write
+			// PrairieGUI.warningLabel("?")
```

Comment:
Maybe a warning if the text hasn't changed -- i.e. if the prompt text is still the default